views/beneficial-owner-gov.html

```diff
@@ -186,16 +200,17 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">Tell us about the government or pu
             classes: "govuk-input--width-10",
             id: "serviceAddressPostcode",
             name: "serviceAddressPostcode",
-            autocomplete: "serviceAddressPostcode"
+            autocomplete: "serviceAddressPostcode",
+            value: serviceAddress.serviceAddressPostcode if serviceAddress
           }) }}
 
           {% endcall %}
         {% endset -%}
 
         {{ govukRadios({
           classes: "govuk-radios",
-          idPrefix: "sameAddress",
-          name: "sameAddress",
+          idPrefix: "isServiceAddressSameAsPrincipalAddress",
+          name: "isServiceAddressSameAsPrincipalAddress",
```

Comment:
Could just have "isSameAddress"

Reply:
i saw that but didn't think it was a good name, isSameAddress as what? so decided to have a name to say x is same as y

Reply:
Yes, that's what I went for with BO individual.